governance/src/main/java/org/apache/servicecomb/governance/handler/ext/RetryExtension.java

```diff
@@ -18,9 +18,21 @@
 package org.apache.servicecomb.governance.handler.ext;
 
 import java.util.List;
+import java.util.stream.IntStream;
 
 public interface RetryExtension {
-  boolean isRetry(List<Integer> statusList, Object result);
+  boolean isRetry(List<String> statusList, Object result);
 
   Class<? extends Throwable>[] retryExceptions();
+
+  default Boolean isContain(List<String> ststusList, String responseStatus) {
```

Comment:
It's better to create a `AbstractRetryExtension` and
1. `isContain` and `judge` give a specific name, e.g. `statusCodeConains`,  `statuceCodeMatch`
2.  give a default implementations for retry and add a default method `String extractStatusCode(Object result)`
3. make `isContain` and `judge`  private

Reply:
done